README.md

```diff
@@ -123,41 +121,33 @@ bin/rails generate component Example title content
       create  app/components/example_component.html.erb
 ```
 
-`ViewComponent` includes template generators for the `erb`, `haml`, and `slim` template engines and will use the template engine specified in your Rails config (`config.generators.template_engine`) by default.
+`ViewComponent` includes template generators for the `erb`, `haml`, and `slim` template engines and will use the template engine specified in the Rails configuration (`config.generators.template_engine`) by default.
 
-If you want to override this behavior, you can pass the template engine as an option to the generator:
+The template engine can also be passed as an option to the generator:
 
 ```bash
 bin/rails generate component Example title content --template-engine slim
-      invoke test_unit
-      create  test/components/example_component_test.rb
-      create  app/components/example_component.rb
-      create  app/components/example_component.html.slim
 ```
 
 #### Implementation
 
-A `ViewComponent` is a Ruby file and corresponding template file (in any format supported by Rails) with the same base name:
+A `ViewComponent` is a Ruby file and corresponding template file with the same base name:
 
 `app/components/test_component.rb`:
 ```ruby
 class TestComponent < ViewComponent::Base
   def initialize(title:)
     @title = title
   end
-
-  private
-
-  attr_reader :title
 end
 ```
 
 `app/components/test_component.html.erb`:
 ```erb
-<span title="<%= title %>"><%= content %></span>
+<span title="<%= @title %>"><%= content %></span>
 ```
 
-We can render it in a view as:
+Which is rendered in a view as:
```

Comment:
Would it make sense to call out the use of the `render` helper here? Maybe "Which is rendered in a view via the `render` helper:"